modules/logging-release-notes-5-9-0.adoc

```diff
@@ -0,0 +1,90 @@
+== Enhancements
+
+[id="logging-release-notes-5-9-0-log-collection"]
+=== Log Collection
+* This enhancement adds the ability to refine the process of log collection by using a workload's metadata and to `drop` or `prune` logs based on their content. Additionally, it allows the collection of infrastructure like journal or container logs and audit logs like `kube api` or `ovn` to only collect individual sources. (link:https://issues.redhat.com/browse/LOG-2155[LOG-2155])
+
+* With this update, the `ClusterLogForwarder` API now supports log forwarding to Azure Monitor Logs, giving users better monitoring abilities. This feature helps users to maintain optimal system performance and streamline log analysis processes in Azure Monitor, which speeds up issue resolution and improves operational efficiency. (link:https://issues.redhat.com/browse/LOG-4605[LOG-4605])
+
+* This enhancement improves collector resource utilization by deploying collectors as a deployment with two replicas. It happens when the only input source defined in the `ClusterLogForwarder` is a receiver input instead of using a daemonset on all nodes. Additionally, collectors deployed like this do not mount the host file system. To use this enhancement, you need to annotate the `ClusterLogForwarder` with `logging.openshift.io/dev-preview-enable-collector-as-deployment`. (link:https://issues.redhat.com/browse/LOG-4779[LOG-4779])
+
+* This enhancement introduces the capability for custom tenant configuration across all supported outputs, facilitating the organization of log records in a logical manner. However, it does not permit custom tenant configuration for {logging} managed storage. (link:https://issues.redhat.com/browse/LOG-4843[LOG-4843])
+
+* With this update, the `ClusterLogForwarder` that specifies an application input with one or more infrastructure namespaces like `default`, `openshift*`, or `kube*`, now requires a service account with the `collect-infrastructure-logs` role. (link:https://issues.redhat.com/browse/LOG-4943[LOG-4943])
+
+* This enhancement introduces the capability for tuning some output settings like compression, retry duration, and max payloads, to match characteristics of the receiver. Additionally, this feature includes a delivery mode to allow administrators to choose between throughput and log durability. For example, `AtLeastOnce` configures minimal disk buffering of collected logs so that the collector can deliver those logs after a restart. (link:https://issues.redhat.com/browse/LOG-5026[LOG-5026])
+
+* This enhancement adds three new Prometheus alerts, warning users about the deprecation of Elasticsearch, Fluentd, and Kibana. (link:https://issues.redhat.com/browse/LOG-5055[LOG-5055])
+
+[id="logging-release-notes-5-9-0-log-storage"]
+=== Log Storage
+* With this update, the validation of the Azure storage secret is now extended to give early warning for certain error conditions. (link:https://issues.redhat.com/browse/LOG-4571[LOG-4571])
+
```

Comment:
* For STS enabled {product-title] 4.14 and later clusters, this update introduces support for short-lived token workload identity federation with Azure and AWS log stores. Local storage requires the addition of a `CredentialMode: static`  annotation under `spec.storage.secret` in the LokiStack CR. (link:https://issues.redhat.com/browse/LOG-4540[LOG-4540])

Reply:
Added this enhancement. Thanks!